docs/contributors/code/how-to-get-your-pull-request-reviewed.md

```diff
@@ -0,0 +1,83 @@
+# How To Get Your Pull Request Reviewed?
+
+Sometimes we publish a Pull Request and no one [reviews](/docs/contributors/repository-management.md#code-review) our work. What to do?
+
+Attracting a review largely isn't about the code – it is about making the reviewing easy.
+
+If you published a Pull Request that isn't getting any comments or reviews, try one of the strategies used by core contributors:
+
+## Create the smallest reasonable PRs
+
+Approving a 2000-line-long PR takes months and feels overwhelming.
+
+Approving a 50-line long PR takes days or hours and feels easy.
+
+Large batches slow you down. Ship your work in small chunks to merge more and learn faster.
+
+## Share relevant context:
+
+Clarify:
+* What problem are you solving?
+* How does your PR solve it?
+* What feedback do you need?
+* What’s out of scope?
+* What’s unintuitive?
+* How to test?
+
+Summarize any related issues and PRs.
+
+It's easier than asking others to go and figure it out.
+
+## Make your PR exciting
+
+All contributions are competing for attention. Make your stand out.
+
+The easiest way? Say why it matters:
+
+❌ A new react hook to get data
+✅ `useEntityRecord`: get data with 10x less boilerplate
+
+Then prove it with code examples, visuals, and screencasts.
+
+## Show your work
+
+Post a link to your PR in related issues & PRs.
+
+Ping commenters of related issues, previous committers, and tech leads.
+
+Bring it up on [WP.org slack](https://make.wordpress.org/core/tag/core-editor-agenda/).
```

Comment:
I'd make it clear that folks can bring these items to meetings (since you link to meeting agenda) rather than just generally wporg slack, although both could be options.

Reply:
Good call @annezazu I just clarified that!